libr/bin/p/bin_elf.inc

```diff
@@ -406,6 +406,11 @@ static RList* entries(RBinFile *bf) {
 		ptr->vaddr = Elf_(r_bin_elf_p2v) (obj, ptr->paddr);
 		ptr->hpaddr = 0x18;  // e_entry offset in ELF header
 		ptr->hvaddr = UT64_MAX;
+
+		if (ptr->vaddr != (ut64)obj->ehdr.e_entry && Elf_(r_bin_elf_is_executable) (obj)) {
+			eprintf ("Entry point could not be determined from file, it was automatically set to 0x%" PFMT64x ".\n", ptr->vaddr);
```

Comment:
make it shorter, i would say:

`Cannot determine entrypoint, using 0x%08"PFMT64x"`